syncmaster/schemas/v1/connections/s3.py

```diff
@@ -2,28 +2,23 @@
 # SPDX-License-Identifier: Apache-2.0
 from typing import Literal
 
-from pydantic import BaseModel, SecretStr, model_validator
+from pydantic import BaseModel, Field, model_validator
 
 from syncmaster.schemas.v1.connection_types import S3_TYPE
+from syncmaster.schemas.v1.connections.connection_base import (
+    CreateConnectionBaseSchema,
+    ReadConnectionBaseSchema,
+    UpdateConnectionBaseSchema,
+)
 
 
 class S3BaseSchema(BaseModel):
```

Comment:
Do we really need `S3BaseSchema` anymore?

Reply:
I also didn't find any usage or need of `from_attributes=True` but left it as is. Let's delete it

Reply:
This may be used for converting SQLAlchemy models to Pydantic models

Reply:
Since we have to combine the auth schema with the connection schema, we don't use such conversion in our connection routes. Therefore, it is not needed at the moment